src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+
+  transform: scale(1);
+  transition: transform 300ms ease-in-out;
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-1;
+    margin: 0;
+    padding-bottom: 4px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .card__name {
+    color: #34568b;
+    transition: transform 300ms ease-in-out;
```

Comment:
The transition property should be set on the default state of the element, not on the hover state. This way, the transition will apply both when the element is hovered over and when the mouse leaves the element, providing a smooth transition in both directions.